chat-plugins/wifi.js

```diff
@@ -149,10 +149,10 @@ var LotteryGiveAway = (function () {
 		this.prize = options.prize;
 		this.maxwinners = options.maxwinners;
 		this.joined = {}; // userid: 0
-
+		var firstPart = this.prize[0];
 		this.reminder = '<center><div class = "broadcast-blue"><font size = 3><b>It\'s giveaway time!</b></font><br/>' +
 			'<font size = 1>Giveaway started by ' + Tools.escapeHTML(host.name) + '</font><br/><br/>' +
-			'<b>' + Tools.escapeHTML(giver.name) + '</b> will be giving away a <b>' + Tools.escapeHTML(this.prize) + '</b>!<br/>' +
+			'<b>' + Tools.escapeHTML(giver.name) + '</b> will be giving away ' + (Number(firstPart) && firstPart !== '1' ? '' : 'a') + ' <b>' + Tools.escapeHTML(this.prize) + '</b>!<br/>' +
```

Comment:
If this.prize is, for example "2 Pikachus", then the message would be "(name) will be giving away 2 Pikachus"
If this.prize's first character is anything else other than a number excluding 1, for example, just "Pikachu", it'd print "(name) will be giving away a Pikachu".
Might need to use string.prototype.trim() before checking though.